src/applications/personalization/view-dependents/components/ViewDependentsHeader/ViewDependentsHeader.jsx

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
 
 const ViewDependentsHeader = () => (
-  <div className="vads-l-row vads-u-padding-bottom--3">
+  <div className="vads-l-row">
     <div className="vads-l-col--12">
       <h1>Your Dependents</h1>
-      <p>
-        If you're getting disability compensation and have a 30% or higher
-        disability rating, you can add a dependent to your disability claim.
-        Adding a dependent to your claim could make you eligible for a higher
-        disability payment.
+      <p className="vads-u-font-size--md vads-u-font-family--serif">
+        Below is the information we have for your dependents. You can file a
+        claim for more disability compensation whenever you add a dependent. If
+        something changes with a dependent's status, you should notify VA.
       </p>
+      <a type="button" className="usa-button-primary va-button-primary">
```

Comment:
According to [MDN's Anchor element docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-type), the `type` attribute on an `<a>` element reads:

> Hints at the linked URL’s format with a MIME type. No built-in functionality.

Since `button` is not included in [MDN's Incomplete list of MIME types]( https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Complete_list_of_MIME_types), was this element supposed to be a `<button>` instead of an `<a>`?

Reply:
removed button type